dev/bots/utils.dart

```diff
@@ -341,6 +341,7 @@ Future<void> runDartTest(String workingDirectory, {
   bool ensurePrecompiledTool = true,
   bool shuffleTests = true,
   bool collectMetrics = false,
+  String? reporter = 'expanded', // TODO-dontmerge-should we just always use --failures-only?
```

Comment:
Is there a pragmatic reason to not use `failures-only`? Should we just remove this parameter and always use `failures-only`?

Reply:
I'm going for it. Added commit [always use --reporter=failures-only](https://github.com/flutter/flutter/pull/150874/commits/2eb30cf46fbed66a8d56342a161fd2980fd8af52)[always use --reporter=failures-only](https://github.com/flutter/flutter/pull/150874/commits/2eb30cf46fbed66a8d56342a161fd2980fd8af52)